doc/run_benchmark.rst

```diff
@@ -191,4 +191,14 @@ mentioned above. Running an arbitrary number of warmup values may also make
 the benchmark less reliable since two runs may use a different number of warmup
 values.
 
+Specializer statistics (`pystats`)
+==================================
 
+If running benchmarks on a CPython built with the ``--enable-pystats`` flag, pyperf will automatically collect statistics from about the bytecode specializer during the run of benchmark code.
```

Comment:
I dislike "will automatically collect statistics". pyperf usually puts everything in the JSON file, but here all it does is to call ``sys._stats_on()`` before and ``sys._stats_off()`` after. It doesn't clear existing statistics in /tmp/py_stats, it doesn't store the in the JSON file, it doesn't compute average or anything.

Would you mind to mention ``sys._stats_on()`` and ``sys._stats_off()`` functions instead?